pkg/service_sync/service_sync.go

```diff
@@ -36,10 +37,18 @@ type ServiceUpdate struct {
 	definitions map[string]types.ServiceInterface
 }
 
-func NewServiceSync(origin string, version string, connectionFactory messaging.ConnectionFactory, handler UpdateHandler) *ServiceSync {
+func getTtl(ttl time.Duration) time.Duration {
```

Comment:
Do you think it is useful to enforce a minimum and safe value for the TTL,
to prevent it from struggling with the default 5s regular updates?

Reply:
Yes, that would make sense. What do you think the minimum should be?

Reply:
Good question. Probably not less than a minute?

Reply:
Done